### PR TITLE
mkosi: install correct package for mypy

### PR DESCRIPTION
**Pull request by teknoraver**
Package "mypy" doesn't exist on all distro, install the correct package depending on the distro.

### CODE REVIEW

File: mkosi.conf.d/05-tools/mkosi.conf.d/fedora.conf [hunk 1]
```diff
@@ -6,4 +6,5 @@ ToolsTreeDistribution=fedora
 [Build]
 ToolsTreePackages=
         gh
+        python3-mypy
```
Comment (yuwata) [inline mkosi.conf.d/05-tools/mkosi.conf.d/fedora.conf]: This seems redundant as crntos-fedora.conf has it.

**Review by DaanDeMeyer — CHANGES_REQUESTED** [review]
You forgot opensuse?

**Comment by teknoraver** [comment]
> You forgot opensuse?

sort of, because their web package search was giving error 504 :)

**Comment by DaanDeMeyer** [comment]
> > You forgot opensuse?
> 
> sort of, because their web package search was giving error 504 :)

pkgs.org is your friend ;)

**Pull request merged**